docs/development/chromium-development.md

```diff
@@ -12,3 +12,64 @@
 - [Discussion Groups](http://www.chromium.org/developers/discussion-groups)
 
 See also [V8 Development](v8-development.md)
+
+# Chromium content development with electron
+
+It is possible to debug chromium content with electron by passing
```

Comment:
All occurrences of `chromium` and `electron` should have the first character capitalized.